LaTeX/LaTeX.sublime-syntax

```diff
@@ -83,17 +83,25 @@ contexts:
       pop: true
     - include: main
 
+  # matches any nospace and pops the context
+  immediate-pop:
+    - match: '(?=\S)'
+      pop: true
+
+  # pops out of the current context if there is a new paragraph
+  paragraph-pop:
+    - match: ^(?=\s*\r?\n)
```

Comment:
We don't need to care about \r in context of syntax definitions as line feeds are normalized by ST. Using `$` instead of `\n` may be needed in case paragraph end needs to be matched at the end of a file as well.

```suggestion
    - match: ^(?=\s*$)
```